stable/traefik/Chart.yaml

```diff
@@ -1,6 +1,6 @@
 apiVersion: v1
 name: traefik
-version: 1.1.0-a
+version: 1.2.0-a
```

Comment:
@krancour I think this chart version is following the version of Traefik, how would you propose we bump the chart version? I can add a "build" tag, e.g. `1.1.0-a+r1`?